airbyte-integrations/connectors/source-snowflake/unit_tests/resource/http/response/response_get_table_with_unknown_type.json

```diff
@@ -0,0 +1,638 @@
+{
+    "resultSetMetaData": {
+        "numRows": 7,
+        "format": "jsonv2",
+        "partitionInfo": [
+            {
+                "rowCount": 7,
+                "uncompressedSize": 6049
+            }
+        ],
+        "rowType": [
+            {
+                "name": "ID",
+                "database": "TEST_AIRBYTE",
+                "schema": "INTEGRATION_TEST",
+                "table": "TEST_TABLE",
+                "nullable": false,
+                "type": "unknown type",
```

Comment:
Can we simplify the response template to only have this row? It would reduce noise and help debug in case of issues

Reply:
Done

Reply:
Done